internal/postprocessor/README.md

```diff
@@ -13,41 +13,75 @@ fork instead of the repo.
+`docker`, rather than with Cloud Build as instructed below, be sure to do so
+from the `google-cloud-go/internal` directory.
 
-### Run post-processor on select clients
+1. In your `google-cloud-go` repo, create a branch.
+2. Make changes to the post-processor.
+3. Test your changes. You can run the post-processor locally on selected
+   clients or on all of the clients in the root directory. If the `branch`
+   flag is not set to a non-empty value, the post-processor will exit early
+   without changes. In the `google-cloud-go/internal/postprocessor` directory:
 
-From the `google-cloud-go/internal/postprocessor` directory run the same command, but with an added `dirs` flag containing a comma-separated list of the names of the clients on which to run the post-processor. The example below shows the command for running the post-processor on the `accessapproval` and `asset` libraries:
-```bash
-go run main.go -client-root="../.." -googleapis-dir="/path/to/local/googleapis" -dirs="accessapproval,asset"
-```
+   ```bash
+   go run . -client-root="../.." -googleapis-dir="/path/to/local/googleapis" -branch="my-branch"
+   ```
+
+   To test only selected clients:
+
+   ```bash
+   go run . -client-root="../.." -googleapis-dir="/path/to/local/googleapis" -branch="my-branch" -dirs="accessapproval,asset"
+   ```
+4. Clean up any changes made by post-processor test runs in the previous step.
+5. Commit your changes.
+6. Open your PR and respond to feedback.
+7. Once your PR is complete (but before merging it!), rebuild the post-processor
+   docker image from your local branch using the
+   `google-cloud-go/internal/cloudbuild.yaml` Cloud Build configuration. In the
+   `google-cloud-go` root directory:
+
+   ```bash
+   gcloud builds submit --project=cloud-devrel-kokoro-resources --config=internal/cloudbuild.yaml
+   ```
+8. Read the SHA of the latest post-processor docker image. In any location:
+
+   ```bash
+   docker pull gcr.io/cloud-devrel-public-resources/owlbot-go:latest
+   docker inspect --format='{{index .RepoDigests 0}}' gcr.io/cloud-devrel-public-resources/owlbot-go:latest
+   ```
+9. In your branch, update the SHA of the post-processor docker image in
+   `google-cloud-go/.github/.OwlBot.lock.yaml`. Commit and push the change to
+   your PR.
+
+   *Note*: OwlBot will eventually open a pull request to update the SHA if it
+   discovers a new version of the container.
+10. After your PR is approved and CI is green, merge your changes.
 
-### Initializing new modules
-To initialize the `internal/version.go`, `go.mod`, `README.md`, and `CHANGES.md` files in a new module, add the module to the slice in `modconfig.go`. The entry should correspond to the location where the `go.mod` file should be initialized minus the prefix "google-cloud-go/"
+## Initializing new modules
+To initialize the `internal/version.go`, `go.mod`, `README.md`, and `CHANGES.md`
+files in a new module, add the module to the slice in `modconfig.go`. The entry
```

Comment:
This should be updated to reflect that this now happens in config.yaml

Reply:
Fixed, thanks!